cfblog/response.py

```diff
@@ -0,0 +1,56 @@
+__author__ = 'vinay'
+from django.core.exceptions import ValidationError
+from django.http.response import Http404, HttpResponse
+from django.template import TemplateSyntaxError
+
+from .models import CmsPage
+from .settings import CMS_TEMPLATE_DIRS, CMS_AUTH_TEST_FUNC
+from .utils import parse_cms_template
+from .validators import validate_and_get_template
+
+
+def cms_response(template_name,
```

Comment:
This will be called render. The idea is for this render to be drop in replacement of django's render: https://docs.djangoproject.com/en/1.8/topics/http/shortcuts/#render.